scripts/deploy_contract.ts

```diff
@@ -39,30 +39,33 @@ async function main() {
     // Deploy voting contract
     logger.info('🗳️  Starting voting contract deployment...');
     logger.info(`📋 Admin address for voting contract: ${address}`);
-    
-    const deployTx = EasyPrivateVotingContract.deploy(wallet, address).send({ 
-        fee: { paymentMethod: sponsoredPaymentMethod } 
+
+    const deployTx = PrivateVotingContract.deploy(wallet, address).send({
+        from: wallet.getAddress(),
```

Comment:
Is this the recommended way to do this is, per Grego? From what I understand a wallet doesn't have just 1 address associated with it.